npm/cypress-schematic/src/builders/cypress/cypressBuilderOptions.ts

```diff
@@ -5,6 +5,7 @@ export interface CypressBuilderOptions extends JsonObject {
   configFile: string | false
   browser: 'electron' | 'chrome' | 'chromium' | 'canary' | 'firefox' | 'edge' | string
   devServerTarget: string
+  devServerBaseUrl: string
```

Comment:
This option would never be used since it is derived from the dev-server, so I think we can remove this.

Reply:
Good point!

Reply:
A general rule is that if the option is available in the `schema.json` it should be listed here as well, so I'd add back in the `devServerTarget` since you shouldn't have to cast the type in the builder.